naptan_create/create_table.py

```diff
@@ -0,0 +1,54 @@
+from io import BytesIO
+import codecs
+import csv
+import boto3
+from zipfile import ZipFile
+from urllib import urlopen
+
+NAPTAN_STOPS_URL = "http://naptan.app.dft.gov.uk/DataRequest/Naptan.ashx?format=csv"
+REGION = 'eu-west-1'
+
+
+def create_naptan_table():
+    url = urlopen(NAPTAN_STOPS_URL)
+
+    # download and unzip file, save Stops.csv in tmp file
+    try:
+        with ZipFile(BytesIO(url.read())) as zip_file:
+            for contained_file in zip_file.namelist():
+                with open("/tmp/stops_temp_file.csv", "wb") as output:
+                    if contained_file == 'Stops.csv':
+                        for line in zip_file.open(contained_file).readlines():
+                            output.write(line)
+                        break
+    except Exception:
```

Comment:
This is too broad an exception, and the print doesn't give us much value - Let's remove the `try/except`

(Same for the next `try/except` block)

Reply:
done